packages/react-cosmos/src/__tests__/create-loader-element.js

```diff
@@ -0,0 +1,88 @@
+const mockComponentElement = {};
+const mockLoader = () => {};
+const mockPropsProxy = () => {};
+const mockStateProxy = () => {};
+const mockProxy1 = {};
+const mockProxy2 = {};
+const mockProxies = [() => mockProxy1, () => mockProxy2];
+const mockComponentsInput = {};
+const mockFixturesInput = {};
+const mockComponentsOutput = {};
+const mockFixturesOutput = {};
+let mockReact;
+let mockLoadComponents;
+let mockLoadFixtures;
+let createLoaderElement;
+let props;
+
+const init = options => {
+  jest.resetModules();
+  jest.resetAllMocks();
+
+  mockReact = {
+    createElement: jest.fn(() => mockComponentElement),
+  };
+  mockLoadComponents = jest.fn(() => mockComponentsOutput);
+  mockLoadFixtures = jest.fn(() => mockFixturesOutput);
+
+  jest.mock('react', () => mockReact);
+  jest.mock('../components/Loader', () => mockLoader);
+  jest.mock('../components/proxies/StateProxy', () => () => mockStateProxy);
+  jest.mock('../components/proxies/PropsProxy', () => mockPropsProxy);
+  jest.mock('../load-modules', () => ({
+    loadComponents: mockLoadComponents,
+    loadFixtures: mockLoadFixtures,
+  }));
+
+  createLoaderElement = require('../create-loader-element.jsx').default;
```

Comment:
I'm with you, already battled with ESLint and [managed to remove most of them here](https://github.com/react-cosmos/react-cosmos/pull/271/commits/adf592102da5f8b88f13008a355e73fbf7f935be)